scripts/missions/amk/06_An_Errand_The_Professors_Price.lua

```diff
@@ -103,7 +144,8 @@ mission.sections =
     -- Got the Starfruit
     {
         check = function(player, currentMission, missionStatus, vars)
-            return currentMission == mission.missionId and
+            return currentMission >= mission.missionId and
```

Comment:
This seems dangerous since we're allowing status of 0 for any and all missions including or greater than 6